Integrations/QuinC/README.md

```diff
@@ -0,0 +1,276 @@
+## Overview
+---
+Use the QuinC integration to protect against and provide additional visibility into phishing and other malicious email attacks.
+This integration was integrated and tested with version 20190926 of QuinC
+## QuinC Playbook
+---
+For example, you can look at "Accessdata: Dump memory for malicious process" playbook to understand how to use this integration.
+## Configure QuinC on Demisto
+---
+
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
+2. Search for QuinC.
+3. Click __Add instance__ to create and configure a new integration instance.
+    * __Name__: a textual name for the integration instance.
+    * __Server URL with scheme (FQDN or IP address in X.X.X.X format with scheme specified)__
+    * __The token is required to connect to QuinC.__
+    * __Trust any certificate (not secure)__
+    * __Use system proxy settings__
+4. Click __Test__ to validate the URLs, token, and connection.
+## Commands
+---
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+1. accessdata-legacyagent-get-processlist
+2. accessdata-legacyagent-get-memorydump
+3. accessdata-read-casefile
+4. accessdata-jobstatus-scan
+5. accessdata-get-jobstatus-processlist
+6. accessdata-get-jobstatus-memorydump
+### accessdata-legacyagent-get-processlist
+---
+Return list of process from legacy agent
+##### Base Command
+
+`accessdata-legacyagent-get-processlist`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| caseid | ID of case | Required | 
+| target_ip | IP address of agent | Required | 
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Accessdata.Job.ID | string | ID of job | 
+| Accessdata.Job.CaseID | string | Case ID | 
+| Accessdata.Job.CaseJobID | string | Concatenated CaseID and JobID (like "1_800") | 
+
+
+##### Command Example
+`accessdata-legacyagent-get-processlist caseid=1 target_ip=X.X.X.X`
+
+##### Context Example
+```
+{
+    "Accessdata.Job": {
+        "ID": 992, 
+        "Type": "Volatile", 
+        "CaseID": "1", 
+        "State": "Unknown", 
```

Comment:
it seems that this command returns State and Type also, but you don't declared them in **Context Outputs**. is there a reasons for that?